app/partials/analysis/tools/powercalc/main.jade

```diff
@@ -0,0 +1,557 @@
+div(ng-controller="powercalcMainCtrl as mainArea")
+  div.lead.bd-danger(ng-show='mainArea.selectedAlgorithm == "Select"')
+    h1 <b>Power Analysis Explanation</b>
+    br
+    h4 <b>CI for One Proportion</b>
+    p <b><i> sample-size determination for estimating a proportion to within a specified margin of error, for either a finite population of specified size, or an infinite population </i></b>
+    input(type='button' style='font-size:20px; margin-bottom:80px' value='CI for One Proportion' ng-click='mainArea.update_algo($event)')
```

Comment:
same as in the controller: Angular two-way binding updates your scope variables for you, so you should use [`ng-model`](https://docs.angularjs.org/api/ng/directive/ngModel) here and similar situations instead of the `event.target`